src/product-card.css

```diff
@@ -0,0 +1,96 @@
+.product-card {
+  width: 166px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card_icon {
+  display: flex;
+  justify-content: center;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.product-card_title {
+  margin: 0 auto 4px;
+  font-family: Roboto, sans-serif;
```

Comment:
Shouldn't repeat this one, just add to `.card` and remove from all child elements